style/git_and_github.qmd

```diff
@@ -1,54 +1,67 @@
 # Using Git and GitHub
 
-* Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
-* All code should be committed via pull request to main. **Never** push to main.
+-   Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
+-   All code should be committed via pull request (PR) to `main`
+
+::: {.callout-important collapse="false" appearance="default" icon="true"}
+**Never** push to `main`!
+:::
 
 Workflow for writing code with Git and GitHub:
 
-* If you haven't already, file an issue that describes what you are doing- whether it be fixing a bug, adding a feature,
-  or something else
-* Make a new branch to work on that feature. GitHub offers a button to do this automatically on the issue page, on the
-  right-hand side (see image)
-* Work on the branch, pushing your code regularly to GitHub so it can be run and inspected when you are not around
-* When it's ready merge the branch to main. If you used the create branch button on GitHub this will automatically close
-  the issue. If the issue is not automatically closed, then close it manually
+## Create an issue
+
+-   if you haven't already, file an issue that describes what you are doing - whether it be fixing a bug, adding a feature, or something else
+-   ensure that the issue is as detailed as possible as it may need to be picked up by someone else. Even if only you intend to work on the issue, over time it can be easy to forget things if there isn't enough information
+
+::: aside
+Issues are a good record to explain what a PR is for. It's also a useful area for input from other members of the team.
+:::
+
+## Create a branch
+
+-   make a new branch to work on that feature. GitHub offers a button to do this automatically on the issue page, on the right-hand side 
+
+[![Screenshot links to specific GitHub help page](branch_img.png){fig-alt="Screenshot of GitHub with the hyperlink to Create a branch highlighted under the category of Development."}](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue)
+
+-   work on the branch, pushing your code regularly to GitHub so it can be run and inspected when you are not around
+-   when it's ready, merge the branch to `main`
+-   delete the branch and close the issue
+
+::: aside
+If you used the `create branch` button on GitHub it will automatically close the issue when merged.
+:::
+
+## Reviewing a PR
+
+For repositories where you are working collaboratively, you must do two things:
```

Comment:
> working collaboratively
this suggests that there is a workflow where we would be happy with people ignoring this entire document?